src/Components/Quill/QuillControlledEditor.tsx

```diff
@@ -233,6 +234,9 @@ export const QuillControlledEditor: React.FC<QuillControlledEditorProps> = ({onS
             {onSave && <Grid xs={12} style={{margin: '0% 1%'}}>
                 <Button disabled={disabled} fullWidth color='primary' variant='contained' onClick={onClickedSave}>Save Feedback</Button>
             </Grid>}
+            {!_.isNil(isDirty) && <Grid xs={12} style={{margin: '0% 1%'}}>
```

Comment:
if is dirty isn't passed we hide the button?

Reply:
Yeah, there's `isDirty` which is only used if we're in a RHF and there's `onSave` which is the old behavior that will show a different button (line 235)